rpc/cloudagent/compute/galleryimage/moc_cloudagent_galleryimage.proto

```diff
@@ -77,6 +77,7 @@ message GalleryImage {
 	ImageSource sourceType = 13;
 	HyperVGeneration hyperVGeneration = 14;
 	CloudInitDataSource cloudInitDataSource = 15;
+	string sourceVM = 16;
```

Comment:
Do we really need this field? We would be storing the path of OS disk in `sourcePath` and ImageSource in `sourceType`. Does this field add any value after the VM is deleted?